modules/Microsoft.ContainerRegistry/registries/readme.md

```diff
@@ -69,6 +69,9 @@ Azure Container Registry is a managed, private Docker registry service based on
 | `systemAssignedIdentity` | bool | `False` |  | Enables system assigned managed identity on the resource. |
 | `tags` | object | `{object}` |  | Tags of the resource. |
 | `trustPolicyStatus` | string | `'disabled'` | `[disabled, enabled]` | The value that indicates whether the trust policy is enabled or not. |
+| `azureADAuthenticationAsArmPolicyStatus` | string | `'enabled'` | `[disabled, enabled]` | The value that indicates whether the Azure AD authentication as ARM policy is enabled or not. |
```

Comment:
Please generate the readme using the CARML [Set-ModuleReadMe.ps1](https://github.com/Azure/ResourceModules/blob/main/utilities/tools/Set-ModuleReadMe.ps1)

You can point it at the module folder:

```pwsh
(Get-ChildItem -Path "$pathToRepository\modules\Microsoft.ContainerRegistry\registries" -Recurse -Filter 'deploy.bicep').FullName |
    ForEach-Object { Set-ModuleReadMe -TemplateFilePath $_ }
```

Reply:
done